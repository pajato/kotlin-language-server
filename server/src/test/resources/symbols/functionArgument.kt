package symbols

fun main(args: Array<String> = arrayOf("a", "b", "c")) {}
