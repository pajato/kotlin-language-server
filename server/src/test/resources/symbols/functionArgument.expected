DocumentSymbol("||path-to-symbols||/functionArgument.kt", File, KtFile, package symbols@@fun main(args: Array<String> = arrayOf("a", "b", "c")) {}@)
    DocumentSymbol("symbols", Package, KtPackageDirective, package symbols, symbols)
        DocumentSymbol("symbols", Package, KtNameReferenceExpression, symbols)
    DocumentSymbol("main", Function, KtNamedFunction, fun main(args: Array<String> = arrayOf("a", "b", "c")) {})
        DocumentSymbol("(args: Array<String> = arrayOf("a", "b", "c"))", Function, KtParameterList, (args: Array<String> = arrayOf("a", "b", "c")))
            DocumentSymbol("args", Function, KtTypeArgumentList, <String>)
                DocumentSymbol("String", Function, KtTypeProjection, String)
            DocumentSymbol("args", Function, KtCallExpression, arrayOf("a", "b", "c"))
                DocumentSymbol("arrayOf", Function, KtNameReferenceExpression, arrayOf)
                DocumentSymbol("args", Function, KtValueArgumentList, ("a", "b", "c"))
        DocumentSymbol("main", Function, KtBlockExpression, {})
